lib/plaid.js

```diff
@@ -3,7 +3,23 @@
 var environments = require('./plaidEnvironments.js');
 var Client = require('./PlaidClient.js');
 
+var isPlaidError = function(err) {
+  const errorTypes = [
```

Comment:
this makes me a little nervous, as it pins our API error types to versioning in the client, which we should probably avoid.

probably best to just leave out this validation altogether.